metricflow/configuration/constants.py

```diff
@@ -16,8 +16,3 @@
 CONFIG_MODEL_PATH = "model_path"
 CONFIG_DWH_HTTP_PATH = "dwh_http_path"
 CONFIG_DWH_ACCESS_TOKEN = "dwh_access_token"
-CONFIG_DBT_REPO = "dbt_repo"
-CONFIG_DBT_PROFILE = "dbt_profile"
-CONFIG_DBT_TARGET = "dbt_target"
```

Comment:
So I'm removing all of the legacy dbt integration locally as well. We will need these three in the future. I'm fine to add them back later, but this is kind of a bummer that we both did this annoying "find all the random integration points and remove them" work.